bentoml/server/bento_api_server.py

```diff
@@ -128,7 +128,7 @@ def healthz_view_func():
         Health check for BentoML API server.
         Make sure it works with Kubernetes liveness probe
         """
-        return Response(response="\n", status=200, mimetype="application/json")
+        return Response(response="\n", status=200, mimetype="text/plain")
```

Comment:
Is there an issue associated with this? Or some documentation?

Reply:
Nope, but I mentioned it in #491 discussion

Reply:
@kautukkundan @flosincapite You can find out more about Kubernetes liveness/readiness at https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Kubernetes checks for return code, `mimetype` and `response` are ignored.